src/psyclone/psyir/transformations/omp_loop_trans.py

```diff
@@ -294,24 +294,35 @@ def _directive(self, children, collapse=None):
             node.reprod = self._reprod
         return node
 
-    def apply(self, node, options=None):
+    def apply(self, node, options=None,
+              reprod: bool = None,
+              **kwargs):
         '''Apply the OMPLoopTrans transformation to the specified PSyIR Loop.
 
-        :param node: the supplied node to which we will apply the \
+        :param node: the supplied node to which we will apply the
                      OMPLoopTrans transformation
         :type node: :py:class:`psyclone.psyir.nodes.Node`
-        :param options: a dictionary with options for transformations\
+        :param bool reprod: indicating whether reproducible reductions should
```

Comment:
`reprod` is now documented twice in this docstring :-)

Reply:
My mistake - I think I was testing something and didn't remove it.